packages/ketcher-core/src/domain/serializers/serializers.types.ts

```diff
@@ -13,8 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  ***************************************************************************/
+import { Struct, Vec2 } from 'domain/entities';
 
 export interface Serializer<T> {
   deserialize: (content: string) => T;
   serialize: (struct: T) => string;
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export interface KetFileNode<T = any> {
+  type: string;
+  fragment?: Struct;
```

Comment:
I don't see usage of fragment field. And usually fragment field is used for identifying that several atoms/anything else are the part of the same structure. Can we remove this field? Or maybe rename?

Reply:
We cannot, it is being used for bonds\atoms